foxai/metrics/metrics.py

```diff
@@ -1,10 +1,11 @@
+from enum import Enum
```

Comment:
Quite a lot of changes to this file were introduced in my PR (fixing quite a lot of issues): https://github.com/softwaremill/FoXAI/pull/92. We could limit this PR to only changes related to versions, and we can append these changes to the metric fixing PR.